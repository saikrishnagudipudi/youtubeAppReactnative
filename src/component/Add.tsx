import {Text, View} from 'react-native';
import React, {Component} from 'react';
class Add extends Component {
  render() {
    return (
      <View>
        <Text>Add</Text>
      </View>
    );
  }
}

export default Add;
