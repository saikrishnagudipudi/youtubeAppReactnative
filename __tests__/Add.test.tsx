import React from 'react';
import renderer from 'react-test-renderer';
import Add from '../src/component/Add'; 
describe('Add Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Add />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
