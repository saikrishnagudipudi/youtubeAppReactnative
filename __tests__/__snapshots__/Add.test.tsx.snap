// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Component renders correctly 1`] = `
<View>
  <Text>
    Add
  </Text>
</View>
`;
